app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
   belongs_to :users
```

Comment:
- [ ] Kindly refactor the Like model to belong to an `author`, not `users`, and also refer to the **User** class.

- [ ] Kindly ensure the appropriate database schema column is renamed accordingly.